File: app/src/main/java/com/lpc/test/bean/Person.java
package com.lpc.test.bean;

/**
 * @ Author     ：v_lipengcheng
 * @ Date       ：Created in 17:06 2019-09-11
 * @ Description：
 */
public abstract class Person {

    public Person() {
        parseParams();
    }

    protected abstract void parseParams();
}

File: kotlin-lib/src/test/java/com/lpc/test/kotlin_lib/Person.kt
package com.lpc.test.kotlin_lib

/**
 * @ Author     ：v_lipengcheng
 * @ Date       ：Created in 15:21 2020-01-16
 * @ Description：
 */
class Person(val id: Int, var name: String) {

    val isLpc: Boolean
        get() = "lpc".equals(name)

    fun isFirst(): Boolean {
        return id == 1
    }

    override fun toString(): String {
        return "Person(id=$id, name='$name')"
    }


}